cephfs/admin/fsadmin.go

```diff
@@ -3,18 +3,15 @@
 package admin
 
 import (
-	"encoding/json"
 	"strconv"
 
+	"github.com/ceph/go-ceph/internal/commands"
 	"github.com/ceph/go-ceph/rados"
 )
 
 // RadosCommander provides an interface to execute JSON-formatted commands that
 // allow the cephfs administrative functions to interact with the Ceph cluster.
-type RadosCommander interface {
-	MgrCommand(buf [][]byte) ([]byte, string, error)
-	MonCommand(buf []byte) ([]byte, string, error)
-}
+type RadosCommander = commands.RadosCommander
```

Comment:
The interface was previously exported by the cephfs/admin module and just taking it away would break api compatibility. Rather than copy the interface code, I thought that an alias would reduce code duplication without breaking callers.